src/Angor/Client/Pages/Wallet.razor

```diff
@@ -1255,4 +1269,48 @@ else
     {
         isShowExtraWord = !isShowExtraWord;
     }
+    
+    
+
+    private void ShowRawTransactionJson(AddressInfo addressInfo)
+    {
+        rawTransactionJson = GetRawTransactionJson(addressInfo);
+        showRawTransactionModal = true;
+    }
+
+    
+    private string GetRawTransactionJson(AddressInfo addressInfo)
+    {
+        var options = new JsonSerializerOptions
+        {
+            WriteIndented = true
+        };
+        return JsonSerializer.Serialize(addressInfo, options);
+    }
+
 }
+
+@if (showRawTransactionModal)
```

Comment:
Is this location for the html by convention?

Reply:
Responding a bit late haha, happy to be back!

I took the convention from here:
https://blog.samferree.dev/basics-in-blazor-modal 
if you prefer, I can move it to the top of the file or create a component for the modal (it can be much better if you can think of another place we could use the component) .